### ISSUE TITLE
Windows `ENVRUNNER` issue

### ISSUE THREAD

**Issue by palmhjell** [newville/pyshortcuts#37]
Hi, this has been a really great find for a project I'm working on (and hopefully many more to come). It has worked perfectly for me on MacOS, but the two Windows machines I've tried it on have had issues with the `ENVRUNNER` set up. Specifically, it seems like the `%~dp0%` in the `call` line is unnecessary; removing that from the `.bat` file makes the shortcut work. So, the `ENVRUNNER` for them ends up looking like this:

```python
ENVRUNNER = """
@ECHO OFF
call activate {0}
echo # run in conda environment "%CONDA_DEFAULT_ENV%":
echo # %*
start %*
""".format(conda_env)
```

and then works as intended, presumably since the file is already running from the conda env directory that is called by the `%~dp0%` and thus only really needs the `activate {env}` command. This might be due to a new Windows/conda update? But I figured I should mention it, since I'm 2 for 2 on that edit working on Windows.

**Comment by newville** [comment]
@palmhjell I think the `%~dp0%` may be needed if that `activate` is not found anywhere in the PATH, or if a default conda environment is enabled by default, but you want to run the script in the dedicated (non-default) environment it was installed in. 
That is, someone might want to *not* presume that the batch file is already running from the desired conda environment.

Does the `%~dp0%` cause problems for you?

**Comment by maphew** [comment]
it should only be `%~dp0`, only a single percent (`%`) for parameter extensions, unlike variables which need one at each end (c.f. https://ss64.com/nt/syntax-args.html).

**Comment by palmhjell** [comment]
@newville This makes sense. In my usage, the shortcut is typically made while in a conda environment as part of a post-install command, and the batch file is located in that environment directory's Scripts location. Dropping the `%~dp0%` makes it work while keeping it causes the shortcut to fail.

@maphew  I just tested the `%~dp0 ` change and it unfortunately did not seem to help given the above situation.

Right now I have a small part in my script to build the shortcut that locates the envrunner file and removes the `%~dp0%` section, and this has worked for the two Windows machines I've tested it on. So, this isn't urgent for me, but it does still seem like something is off here.

(FYI, I work on a mac so I'm pretty useless here besides reporting what has worked and what hasn't, but happy to test things as needed...)
